website/docs/script/js/js-vm.mdx

```diff
@@ -0,0 +1,397 @@
+---
+id: js-vm
+title: "CKB JS VM"
```

Comment:
title seems too broad. is there a more specific title to describe what this article is for?